src/windows/disk.rs

```diff
@@ -383,6 +374,63 @@ fn os_string_from_zero_terminated(name: &[u16]) -> OsString {
     OsString::from_wide(&name[..len])
 }
 
+fn get_disk_kind(device_path: &[u16], borrowed_handle: &Option<HandleWrapper>) -> DiskKind {
```

Comment:
```suggestion
fn get_disk_kind(device_path: &[u16], borrowed_handle: Option<&HandleWrapper>) -> DiskKind {
```

you can use `.as_ref()` on the `Option` for that.

Reply:
Done